web/package.json

```diff
@@ -24,6 +25,7 @@
     "d3": "7.3.0",
     "d3-dag": "0.9.1",
     "date-fns": "2.28.0",
+    "debug": "4.3.4",
```

Comment:
Probably will remove this package after fixing all the server issues.

Reply:
Will this be removed now or are we keeping it?

Reply:
Removed @mathnogueira 👍